src/main/scala/uk/gov/nationalarchives/consignmentexport/BagMetadata.scala

```diff
@@ -26,7 +26,7 @@ class BagMetadata(keycloakClient: KeycloakClient)(implicit val logger: SelfAware
   }
 
   private def getConsignmentDetails(consignment: GetConsignment, exportDatetime: ZonedDateTime): Map[String, Option[String]] = {
-    val seriesCode: Option[String] = consignment.series.map(_.code)
+    val seriesCode: Option[String] = Some(consignment.series.map(_.code).getOrElse(""))
```

Comment:
I think that maybe this should be stricter, and that only if the consignment type is `judgment` then the series should be blank. We would still want to catch instance for standard transfers where the series is missing, at least for the time being.